src/components/AddressSearch.js

```diff
@@ -157,14 +157,13 @@ const AddressSearch = (props) => {
                         setDisplayListViewBorder(false);
                     }}
                     query={{
-                        key: 'AIzaSyC4axhhXtpiS-WozJEsmlL3Kg3kXucbZus',
```

Comment:
This API key became unused because we are ALWAYS proxying the request through our backend server